spacy/lang/tr/tokenizer_exceptions.py

```diff
@@ -1,119 +1,184 @@
+    {ORTH: "Ütğm.", LEMMA: "üsteğmen", NORM:  "üsteğmen"},
+    {ORTH: "vb.", LEMMA: "ve benzeri"},
+    {ORTH: "vs.", LEMMA: "vesaire", NORM: "vesaire"},
+    {ORTH: "Yard.", LEMMA: "yardımcı", NORM: "yardımcı"},
+    {ORTH: "Yar.", LEMMA: "yardımcı", NORM: "yardımcı"},
+    {ORTH: "Yd.Sb.", LEMMA: "yedek subay"},
+    {ORTH: "Yard.Doç.", LEMMA: "yardımcı doçent"},
+    {ORTH: "Yar.Doç.", LEMMA: "yardımcı doçent"},
+    {ORTH: "Yb.", LEMMA: "yarbay", NORM: "yarbay"},
+    {ORTH: "Yrd.", LEMMA: "yardımcı", NORM: "yardımcı"},
+    {ORTH: "Yrd.Doç.", LEMMA: "yardımcı doçent"},
+    {ORTH: "Y.Müh.", LEMMA: "yüksek mühendis"},
+    {ORTH: "Y.Mim.", LEMMA: "yüksek mimar"},
+    {ORTH: "yy.", LEMMA: "yüzyıl", NORM: "yüzyıl"},
+]
 
-for orth in ["Dr.", "yy."]:
-    _exc[orth] = [{ORTH: orth}]
+for abbr in _abbr_period_exc:
+    _exc[abbr[ORTH]] = [abbr]
 
+_abbr_exc = [
+    {ORTH: "AB", LEMMA: "Avrupa Birliği", NORM: "Avrupa Birliği"},
+    {ORTH: "ABD", LEMMA: "Amerika Birleşik Devletleri", NORM: "Amerika"},
+    {ORTH: "ABS", LEMMA: "fren", NORM: "fren"},
+    {ORTH: "AOÇ", LEMMA: "Atatürk orman çiftliği"},
+    {ORTH: "ASKİ", LEMMA: "Ankara su ve kanalizasyon idaresi"},
+    {ORTH: "Bağ-kur", LEMMA: "Bağkur", NORM: "Bağkur"},
+    {ORTH: "BDDK", LEMMA: "bankacılık düzenleme ve denetleme kurumu"},
+    {ORTH: "BJK", LEMMA: "Beşiktaş", NORM: "Beşiktaş"},
+    {ORTH: "ESA", LEMMA: "Avrupa uzay ajansı", NORM: "Avrupa uzay ajansı"},
+    {ORTH: "FB", LEMMA: "Fenerbahçe", NORM: "Fenerbahçe"},
+    {ORTH: "GATA", LEMMA: "Gülhane askeri tıp akademisi"},
+    {ORTH: "GS", LEMMA: "Galatasaray", NORM: "Galatasaray"},
+    {ORTH: "İSKİ", LEMMA: "İstanbul su ve kanalizasyon idaresi"},
+    {ORTH: "KBB", LEMMA: "kulak burun boğaz"},
+    {ORTH: "RTÜK", LEMMA: "radyo ve televizyon üst kurulu", NORM: "radyo ve televizyon üst kurulu"},
+    {ORTH: "TBMM", LEMMA: "Türkiye Büyük Millet Meclisi"},
+    {ORTH: "TC", LEMMA: "Türkiye Cumhuriyeti"},
+    {ORTH: "TÜİK", LEMMA: "Türkiye istatistik kurumu", NORM: "Türkiye istatistik kurumu"},
+    {ORTH: "YÖK", LEMMA: "Yüksek Öğrenim Kurumu"},
+]
+
+for abbr in _abbr_exc:
+    _exc[abbr[ORTH]] = [abbr]
+
+
+
+_num = r"[+-]?\d+([,.]\d+)*"
+_ord_num = r"(\d+\.)"
+_date = r"(((\d{1,2}[./-]){2})?(\d{4})|(\d{1,2}[./]\d{1,2}(\.)?))"
+_dash_num = r"(([{al}\d]+/\d+)|(\d+/[{al}]))".format(al=ALPHA)
+_roman_num =  "M{0,3}(?:C[MD]|D?C{0,3})(?:X[CL]|L?X{0,3})(?:I[XV]|V?I{0,3})"
+_roman_ord = r"({rn})\.".format(rn=_roman_num)
+_time_exp = r"\d+(:\d+)*"
+
+_inflections = r"'[{al}]+".format(al=ALPHA_LOWER)
+_abbrev_inflected = r"[{a}]+\.'[{al}]+".format(a=ALPHA, al=ALPHA_LOWER)
+
+_nums = r"(({d})|({dn})|({te})|({on})|({n})|({ro})|({rn}))({inf})?".format(d=_date, dn=_dash_num, te=_time_exp, on=_ord_num, n=_num, ro=_roman_ord, rn=_roman_num, inf=_inflections)
 
 TOKENIZER_EXCEPTIONS = _exc
+TOKEN_MATCH = re.compile(r"^({u})|({abbr})|({n})$".format(u=URL_PATTERN, n=_nums, abbr=_abbrev_inflected)).match
```

Comment:
If you want to support splitting trailing commas or parentheses from URLs, it would be better to have the URL pattern as `URL_MATCH` (you can just use the default `URL_MATCH` setting) and only have the other tokens as part of `TOKEN_MATCH`.

Reply:
Thanks for pointing out, I made the corresponding change and removed the `URL_PATTERN` from the custom match.